streams/src/main/java/org/apache/kafka/streams/errors/BrokerNotFoundException.java

```diff
@@ -23,6 +23,7 @@
  * @see org.apache.kafka.streams.StreamsConfig
  */
 @SuppressWarnings("unused")
+@Deprecated(since = "4.2", forRemoval = true)
```

Comment:
```suggestion
@Deprecated
```

Reply:
We usually don't add metadata to the tag, but use JavaDocs for this.

Reply:
ok, I followed some of the existing usages like `RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter` 
Should I remove it?

Reply:
Interesting. `RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter` is an internal class though (note, it's in package `internals`), and it not user facing.

We usually just use plain `@Deprecated` (for example, compare `JoinWindows`).